tests/mitmproxy_addons/callback.py

```diff
@@ -32,38 +47,64 @@ def __init__(self):
 
     def reset(self):
         self.config = {
-            "callback_url": "",
+            "callback_request_url": "",
+            "callback_response_url": "",
             "filter": None,
         }
 
     def load(self, loader):
         loader.add_option(
             name="callback",
             typespec=dict,
-            default={"callback_url": "", "filter": None},
+            default={
+                "callback_request_url": "",
+                "callback_response_url": "",
```

Comment:
seems like the options are, well, optional (we use `config.get()` rather than `config[]`) -- so maybe just omit these?

Reply:
They are optional but in absence of having `__init__` set all the properties, I've tried to set them all in `load`, so readers can see what properties the addon uses. They aren't strictly required though as you say.